code/modules/assembly/flash.dm

```diff
@@ -71,11 +71,13 @@
 /obj/item/device/assembly/flash/proc/AOE_flash(bypass_checks = FALSE, range = 3, power = 5, targeted = FALSE, mob/user)
 	if(!bypass_checks && !try_use_flash())
 		return FALSE
-	var/list/mob/targets = get_flash_targets(get_turf(src), range, FALSE)
+  
+	var/list/targets = get_flash_targets(get_turf(src), range, FALSE)

```

Comment:
This change does nothing. New 512 features let us access values of list members iirc and that's all the old format would allow, it didn't filter.

Reply:
I think list/mob/targets is dumb anyways :p

Reply:
A pr to just change coding style generally isn't allowed.

Reply:
We've had PRs explicitly to change 0 to FALSE which is code-wise equivalent but stylistically different to better represent what was occurring and fit with standards. If the first part of the review is as you say it is, I don't see why we can't take the logic of 0/FALSE PRs and apply it here.

Since `mob` is irrelevant, removing it better represents that the list has no unique functionality from other lists in code (which is at least what I thought it was doing until you said otherwise). TBH this is literally the only time I have seen a code typed further than list, otherwise I'd agree i'm just being pedantic.

Reply:
I meant prs to change style that isn't enforced by guidelines.